docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+!build/
```

Comment:
I noticed that there's already a `.gitignore` file in the root directory of this repository. You should just add the following line to it:
```text
docs/build/
```
You don't need a `!` in front. You can delete this file (`docs/.gitignore`).